cirq/google/engine/engine_test.py

```diff
@@ -148,34 +149,39 @@
         '1562544000021',
         'metrics': [{
             'name': 'xeb',
-            'targets': ['q0_0', 'q0_1'],
+            'targets': ['0_0', '0_1'],
             'values': [{
                 'doubleVal': .9999
             }]
         }, {
             'name': 'xeb',
-            'targets': ['q0_0', 'q1_0'],
+            'targets': ['0_0', '1_0'],
             'values': [{
                 'doubleVal': .9998
             }]
         }, {
             'name': 't1',
-            'targets': ['q0_0'],
+            'targets': ['0_0'],
             'values': [{
                 'doubleVal': 321
             }]
         }, {
             'name': 't1',
-            'targets': ['q0_1'],
+            'targets': ['0_1'],
             'values': [{
                 'doubleVal': 911
             }]
         }, {
             'name': 't1',
-            'targets': ['q1_0'],
+            'targets': ['1_0'],
             'values': [{
                 'doubleVal': 505
             }]
+        }, {
+            'name': 'temp',
```

Comment:
Consider naming this something like "globalMetric" to clarify its usage?

Reply:
done